cli/js/web/blob.ts

```diff
@@ -164,17 +164,14 @@ async function readBytes(
 
 // A WeakMap holding blob to byte array mapping.
 // Ensures it does not impact garbage collection.
-export const blobBytesWeakMap = new WeakMap<domTypes.Blob, Uint8Array>();
+export const blobBytesWeakMap = new WeakMap<Blob, Uint8Array>();
 
-export class DenoBlob implements domTypes.Blob {
+export class DenoBlob implements Blob {
```

Comment:
Do we want to use `DenoX` or `XImpl`?  Seems like we should be consistent.  I don't personally care, but I did `XImpl` in the other PR.

Reply:
Yes, XImpl is preferable. I should have fixed it here.